src/test/java/uk/gov/ons/ctp/response/collection/exercise/service/validator/MandatoryEventValidatorTest.java

```diff
@@ -1,9 +1,8 @@
 package uk.gov.ons.ctp.response.collection.exercise.service.validator;
 
 import static org.hamcrest.CoreMatchers.instanceOf;
-import static org.junit.Assert.assertFalse;
-import static org.junit.Assert.assertThat;
-import static org.junit.Assert.assertTrue;
+import static org.junit.Assert.*;
```

Comment:
This might be one for a proper java dev but i've always been told it's better to import specific functions instead of them all 🎨